apps/src/publicKeyCryptography/NumberedSteps.jsx

```diff
@@ -17,15 +17,38 @@ export default function NumberedSteps(props) {
   return (
     <table>
       <tbody>
-        {React.Children.map(props.children, (child, index) => (
-          <tr>
-            <td style={style.td}>{(index + 1) + ')'}</td>
-            <td style={style.td}>{child}</td>
-          </tr>
-        ))}
+        {React.Children.map(props.children, (child, index) => {
+          const stepRequirements = child.props && child.props.requires || [];
```

Comment:
Because I wanted to apply the opacity to the whole table row. I could
probably render the row from step though.

On Aug 31, 2016 5:39 PM, "Elijah Hamovitz" <notifications@github.com> wrote:

> In apps/src/publicKeyCryptography/NumberedSteps.jsx
> <https://github.com/code-dot-org/code-dot-org/pull/10406#discussion_r77098298>
> :
>
> > @@ -17,15 +17,38 @@ export default function NumberedSteps(props) {
> >    return (
> >      <table>
> >        <tbody>
> > -        {React.Children.map(props.children, (child, index) => (
> > -          <tr>
> > -            <td style={style.td}>{(index + 1) + ')'}</td>
> > -            <td style={style.td}>{child}</td>
> > -          </tr>
> > -        ))}
> > +        {React.Children.map(props.children, (child, index) => {
> > +          const stepRequirements = child.props && child.props.requires || [];
>
> it seems a bit odd that this logic lives here and not in Step; any reason
> why trStyle cannot instead be applied to the div in Step's render method?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/10406/files/96801eb4461c713a313ef7285d291a451e5cd266#r77098298>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABinkUiH7urC4mxLMtwE4GgwDXqXr6Y1ks5qlh7CgaJpZM4JyMvb>
> .
>